### DISCUSSION TITLE
Note Map Feature Idea | Do you guys want draggable notes?

### DISCUSSION THREAD

**Discussion by CobriMediaJulien** [TriliumNext/Trilium#6068] [General]
Hey guys,

how do u use the note map?

I think the notes in note map pull too much at each other. Sometimes i have a lot of notes in one bulk and if i drag one note out of this mess of notes everything just gets pulled towards the note. So i can´t seperate the notes in note map view.

How do u use the note map or do u have a work around?

![image](https://github.com/TriliumNext/Notes/assets/56324041/39f443b9-b972-41c4-b8e0-3794127c3d34)

I´m testing right now Anytype and their note map. There the notes push each other away. Which is kinda nice for readability.

**Comment by CobriMediaJulien** [comment]
Hey guys. I just implementated it myself. If some guys want to have it too, we can add it to the main repo. It´s just 4 lines of code:

Here a Gif to show how it´s working:

![NoteMap-TriliumNotesand2morepages-Profil1-MicrosoftEdge2024-06-1618-47-45-ezgif com-crop (1)](https://github.com/TriliumNext/Notes/assets/56324041/ddb2f007-0bd1-46af-af90-e50498529edc)

**Reply by maphew** [reply]
Seems like a natural enhancement to me. Create a fork of the main repo, make your code changes to that and submit a Pull Request. Then the core maintainers can review and ensure it doesn't interfere with any current behaviour.

**Comment by CobriMediaJulien** [comment]
https://github.com/zadam/trilium/discussions/4807

i opened a discussion in zadams repo. Maybe somebody can help. I was able to build a exe but the changes i made in the widget javascript file note_map.js aren´t applied in the building process. So the desktop app does not include my code snipped. Would appreciate a discussion there to fix the problem and maybe prepare a first release on this side.
